blockchain/blockchain-wallet-v4-frontend
# Add Language Support

## Description
- Enables hash routing throughout entire app
- Adds ability for wallet to handle language profiles present in URL
- Updates NGINX to pull 3rd party URLs from build pipeline

## Change Type
Feature

## Testing Steps
- URLs such as `[domain]/ru/#/login` should load the wallet in Russian even if preferences have been set to English. 
- Links both to and from wallet still work as designed

## Code Checklist
